### DISCUSSION TITLE
Current Priorities

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5147] [Announcements]
We've been discussing what we want to prioritize for code-server for the next couple of sprints. The overall theme is stability and confidence. Here is what we'll be focusing on: 

1. code coverage - increase to 80%
2. patches - removed, upstreamed or tested via e2e
3. smooth release process
4. [stretch] automate vscode updates

We want code-server to be a project that is easy to maintain and reliable to use. If you'd like to contribute in any of these areas, please reply to this thread and we'd be happy to get you on board.

-the code-server maintainers ✌🏼

**Comment by nisancigokmen** [comment]
well done  🎸

**Comment by nisancigokmen** [comment]
I can join and support the organization :)

**Reply by jsjoeio** [reply]
We can't add you to the organization for security reasons but if you're looking to contribute to something specific, let me know and we can figure something out!

**Comment by nisancigokmen** [comment]
okey,thanks:))
,gökmen.

Joe Previte ***@***.***>, 4 May 2022 Çar, 23:39 tarihinde
şunu yazdı:
